listenstore/listenstore/utils.py

```diff
@@ -3,23 +3,32 @@
 import argparse
 from setproctitle import setproctitle
 import json
-
-
-## TODO read config from whereever
-def config():
-    return {
-        "kafka_server": "localhost:9092",
-        "cassandra_server": "localhost",
-        "cassandra_keyspace": "listenbrainz"
-    }
+import ConfigParser
+import os
+
+def config(opt_vars):
+    """ Parse config file else return default values """
```

Comment:
**Done!**
(This is the minimal what i could create, [https://github.com/pinkeshbadjatiya/listenbrainz-server/blob/040dd4a20fa69e96b29affa15777451d7caffe85/listenstore/listenstore/loadconfig.py](https://github.com/pinkeshbadjatiya/listenbrainz-server/blob/040dd4a20fa69e96b29affa15777451d7caffe85/listenstore/listenstore/loadconfig.py) )